edx_reverification_block/xblock/reverification_block.py

```diff
@@ -29,6 +30,20 @@ def student_view(self, context=None):
         course_id = self.get_course_id()
         item_id = unicode(self.scope_ids.usage_id)
         checkpoint_name = CHECKPOINT_NAME
+        # TODO: How to get the user object from runtime to check the status of that user in the given checkpoint?
+        verification_status = self.runtime.service(self, "reverification").get_status(
+            course_id=course_id,
+            checkpoint_name=checkpoint_name
+        )
+        if verification_status:
+            # TODO: What message will be displayed to user if it is already has any status?
+            frag = Fragment(unicode(verification_status))
+            return frag
+        self.runtime.service(self, "reverification").start_verification(
```

Comment:
I like the idea of having the verification service return the link into the re-verification flow, so that the XBlock doesn't need to assume a particular URL structure.  Could you update this code so that it uses the URL returned by the service.